GsaGH/Parameters/1_Properties/GsaBucklingLengthFactorsGoo.cs

```diff
@@ -8,31 +8,13 @@ namespace GsaGH.Parameters {
   /// Goo wrapper class, makes sure <see cref="GsaBucklingLengthFactors"/> can be used in Grasshopper.
   /// </summary>
   public class GsaBucklingLengthFactorsGoo : GH_OasysGoo<GsaBucklingLengthFactors> {
-    public static string Name => "BucklingLengthFactors";
-    public static string NickName => "fLs";
-    public static string Description => "GSA Buckling Length Factors";
+    public static string Name => "BucklingFactors";
+    public static string NickName => "BFs";
+    public static string Description => "GSA Buckling Factors";
     public override OasysPluginInfo PluginInfo => GsaGH.PluginInfo.Instance;
 
     public GsaBucklingLengthFactorsGoo(GsaBucklingLengthFactors item) : base(item) { }
 
     public override IGH_Goo Duplicate() => new GsaBucklingLengthFactorsGoo(Value);
-
-    public override bool CastFrom(object source) {
-      if (source == null)
-        return false;
-
-      if (base.CastFrom(source))
-        return true;
-
-      if (!GH_Convert.ToDouble(source, out double val, GH_Conversion.Both)) {
```

Comment:
goo without castFrom? it was intentional?

Reply:
Yes I didn’t want anyone to cast from a single double into three doubles, as the factors are unrelated and would never realistically all be the same. So removed the cast